packages/loader/container-loader/src/container.ts

```diff
@@ -341,6 +341,7 @@ export async function ReportIfTooLong(
  * @internal
  */
 export interface IPendingContainerState {
+	attached: true;
```

Comment:
Didn't know you can set values at the interface definition. Is it safe?

Reply:
its not setting a value. its saying the only valid value is true

Reply:
it allows union types like `IPendingContainerState  | IPendingDetachedContainerState` to be discriminated via the `attached` property, such that typescript know that if it is true, it is IPendingContainerState and if it is false it is IPendingDetachedContainerState